package.json

```diff
@@ -7,29 +7,29 @@
   "author": "Shawn Tabrizi <shawntabrizi@gmail.com>",
   "license": "MIT",
   "scripts": {
-		"dev": "./node_modules/.bin/nodemon --exec ./node_modules/.bin/ts-node ./src/index.ts",
-		"start": "./node_modules/.bin/ts-node ./src/index.ts",
-		"update": "././node_modules/.bin/ncu -u && yarn",
-		"lint": "eslint . --ext .ts --fix",
-		"build": "./node_modules/.bin/tsc --pretty"
-	},
-	"devDependencies": {
-		"@babel/cli": "^7.14.5",
-		"@babel/core": "^7.14.6",
-		"@babel/preset-typescript": "^7.14.5",
-		"@types/node": "^16.0.1",
-		"@types/yargs": "^17.0.2",
-		"@typescript-eslint/eslint-plugin": "^4.28.2",
-		"@typescript-eslint/parser": "^4.28.2",
-		"eslint": "^7.30.0",
-		"nodemon": "^2.0.9",
-		"npm-check-updates": "11.8.2",
-		"ts-loader": "^9.2.3",
-		"ts-node": "^10.0.0",
-		"typescript": "4.3.5"
-	},
-	"dependencies": {
-		"@polkadot/api": "4.17.1",
-		"bn.js": "^5.2.0"
-	}
+    "dev": "./node_modules/.bin/nodemon --exec ./node_modules/.bin/ts-node ./src/index.ts",
+    "start": "./node_modules/.bin/ts-node ./src/index.ts",
+    "update": "././node_modules/.bin/ncu -u && yarn",
+    "lint": "eslint . --ext .ts --fix",
+    "build": "./node_modules/.bin/tsc --pretty"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.6",
+    "@babel/preset-typescript": "^7.14.5",
+    "@types/node": "^16.0.1",
+    "@types/yargs": "^17.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
+    "@typescript-eslint/parser": "^4.28.2",
+    "eslint": "^7.30.0",
+    "nodemon": "^2.0.9",
+    "npm-check-updates": "11.8.2",
+    "ts-loader": "^9.2.3",
+    "ts-node": "^10.0.0",
+    "typescript": "4.3.5"
+  },
+  "dependencies": {
+    "@polkadot/api": "5.7.1",
+    "bn.js": "^5.2.0"
+  }
```

Comment:
This just make the file consistent with all spaces instead of a mix of spaces and tabs